src/test/java/com/openlattice/hazelcast/serializers/EntityTypePropertyKeyStreamSerializerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.openlattice.hazelcast.serializers;
+
+import com.kryptnostic.rhizome.hazelcast.serializers.AbstractStreamSerializerTest;
+import com.openlattice.edm.type.EntityTypePropertyKey;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+
+public class EntityTypePropertyKeyStreamSerializerTest extends AbstractStreamSerializerTest<EntityTypePropertyKeyStreamSerializer, EntityTypePropertyKey>
+        implements Serializable {
+private static final long serialVersionUID = -4933403371497497344L;
+
+@Override
+protected EntityTypePropertyKeyStreamSerializer createSerializer() {
+        return new EntityTypePropertyKeyStreamSerializer();
+        }
+
+@Override
+protected EntityTypePropertyKey createInput() {
+        return new EntityTypePropertyKey( UUID.randomUUID(), UUID.randomUUID() );
```

Comment:
I like to throw these in TestDataFactory since it can be useful to have a mock object easily available